ci-operator/step-registry/mirror-images/by-oc-adm/mirror-images-by-oc-adm-commands.sh

```diff
@@ -122,9 +122,26 @@ then
         source "${SHARED_DIR}/mirror-proxy-conf.sh"
 fi
 
-# execute the mirror command
 cmd="oc adm release -a '${new_pull_secret}' mirror ${args[*]} | tee '${mirror_output}'"
-run_command "$cmd"
+
+MAX_ATTEMPTS=5
+ATTEMPTS=0
+SUCCESS=false
+while [ "${SUCCESS}" = false ] && (( ATTEMPTS++ < MAX_ATTEMPTS )); do
```

Comment:
Do you mind to sync the re-tries logic to `mirror-images-by-oc-adm-in-bastion` together ?